src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -1189,76 +1252,143 @@ private static int GetDeclarationDepth(SyntaxTrivia trivia)
 
         private static int GetDeclarationDepth(SyntaxNode? node)
         {
-            if (node != null)
+            if (node is null)
+            {
+                return 0;
+            }
+
+            if (node.IsStructuredTrivia)
+            {
+                var tr = ((StructuredTriviaSyntax)node).ParentTrivia;
+                return GetDeclarationDepth(tr);
+            }
+            else if (node.Parent != null)
             {
-                if (node.IsStructuredTrivia)
+                if (node.Parent.IsKind(SyntaxKind.CompilationUnit))
                 {
-                    var tr = ((StructuredTriviaSyntax)node).ParentTrivia;
-                    return GetDeclarationDepth(tr);
+                    return 0;
                 }
-                else if (node.Parent != null)
-                {
-                    if (node.Parent.IsKind(SyntaxKind.CompilationUnit))
-                    {
-                        return 0;
-                    }
 
-                    int parentDepth = GetDeclarationDepth(node.Parent);
+                int parentDepth = GetDeclarationDepth(node.Parent);
 
-                    if (node.Parent.Kind() is SyntaxKind.GlobalStatement or SyntaxKind.FileScopedNamespaceDeclaration)
-                    {
-                        return parentDepth;
-                    }
+                if (node.Parent.Kind() is SyntaxKind.GlobalStatement or SyntaxKind.FileScopedNamespaceDeclaration)
+                {
+                    return parentDepth;
+                }
 
-                    if (node.IsKind(SyntaxKind.IfStatement) && node.Parent.IsKind(SyntaxKind.ElseClause))
-                    {
-                        return parentDepth;
-                    }
+                if (node.IsKind(SyntaxKind.IfStatement) && node.Parent.IsKind(SyntaxKind.ElseClause))
+                {
+                    return parentDepth;
+                }
 
-                    if (node.Parent is BlockSyntax)
+                if (node.Parent is BlockSyntax)
+                {
+                    return parentDepth + 1;
+                }
+
+                if (node.Parent is InitializerExpressionSyntax ||
+                    node is { Parent: AnonymousObjectMemberDeclaratorSyntax } ||
```

Comment:
I was confused by usage of `node.Parent is ...` on above line but `node is { Parent: ... }` on this line. Was there a reason for the difference? Consider consolidating this condition into a single pattern like

```cs
node is { Parent: InitializerExpressionSyntax or AnonymousObjectMemberDeclaratorSyntax }
    or AssignmentExpressionSyntax { Parent: InitializerExpressionSyntax }
```

Reply:
> Was there a reason for the difference?

There is no reason. During development of the PR these conditions where a bit different, but after changing I just didn't simplify them enough)